cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -328,21 +334,27 @@ def _convert_hostname_to_ip(self, host_name):
         del socket
         return ip
 
-    def get_active_ceph_mgr(self):
+    def get_active_and_standby_mgrs(self):
         monitoring_endpoint_port = self._arg_parser.monitoring_endpoint_port
         monitoring_endpoint_ip = self._arg_parser.monitoring_endpoint
+        standby_mgrs = []
         if not monitoring_endpoint_ip:
-            cmd_json = {"prefix": "mgr services", "format": "json"}
+            cmd_json = {"prefix": "status", "format": "json"}
```

Comment:
With this, I think we need to update the --upgrade flag too right?

Reply:
> With this, I think we need to update the --upgrade flag too right?

IMHO, we don't have to.
We have 3 sets of flags, `common`, 'output` and `upgrade` flags. The current changes we made (to add the standby mgrs also into the list of mgrs) is part of `output` flag, which is independent of `upgrade` flag (and don't affect the `upgrade` flag).
`upgrade` is mostly about upgrading the permissions/caps of the existing user.

Reply:
Ah right, the caps are only needed for the operator, not the script, my bad!